### ISSUE TITLE
Prevent unnecessary focus changes during voiceover. Investigate why this happens. 

### ISSUE THREAD

**Issue by rupalkhilari** [SFSU-YDiOS/YouDescribe-iOS#26]

**Comment by rupalkhilari** [comment]
This happens when the youtube player controls are autohidden - in this case the Voiceover changes focus to the 'show controls button' which is inserted instead of the controls bar. YouTube has removed the 'autohide' parameter so the controls cannot be displayed permanently. Another option is to somehow not allow voiceover from changing focus automatically.
